GIT/Training_jaydippansuriya

```diff
@@ -0,0 +1,77 @@
+1. Which of the following adds all new files to local repository ?
```

Comment:
Update document as per the following format:

Question:
Answer:
Reason:

@jaydip3112